packages/client/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@mattermost/client",
+  "version": "6.7.0-0",
+  "description": "JavaScript/TypeScript client for Mattermost",
+  "keywords": [
+    "mattermost"
+  ],
+  "homepage": "https://github.com/mattermost/mattermost-webapp/tree/master/packages/client#readme",
+  "license": "MIT",
+  "files": [
+    "lib"
+  ],
+  "main": "./lib/index.js",
```

Comment:
Unlike types which uses `exports` because we import its files one-by-one, this package only exports a single file which contains all the types used for it